src/runtime/internal/Component.ts

```diff
@@ -104,8 +104,24 @@ function make_dirty(component, i) {
 	component.$$.dirty[(i / 31) | 0] |= (1 << (i % 31));
 }
 
+function is_svelte_element(component) {
+  while (component) {
+    if (component.constructor === SvelteElement)
+      return true;
+    component = Object.getPrototypeOf(component);
+  }
+  return false;
+}
+
+function find_containing_component(component) {
+  while (component && !is_svelte_element(component)) {
+    component = component.parentNode;
+  }
+  return component;
+}
+
 export function init(component, options, instance, create_fragment, not_equal, props, append_styles, dirty = [-1]) {
```

Comment:
"init" function call in `constructor` of `SvelteElement`, can not get `parentNode`, so this PR can not fix the issue.

you can get `parentNode` in `connectedCallback`.